utils/config.ts

```diff
@@ -0,0 +1,66 @@
+import { Category } from './poco-tools';
+import json from '../config/config.json';
+
+const config = json as Config;
+
+function isNativeChain(chain?: ChainConfig) {
+    chain = chain ?? config.chains.default;
+    return chain.asset === 'Native' || process.env.TEST__IS_NATIVE_CHAIN === 'true';
+}
```

Comment:
I wonder if we should add `function isTokenChain return function !isNativeChain` to be used depending on the context

Reply:
My logic is: our specific case it `native` mode because we can be in that mode only on our specific chain, we are in `token` mode for all the other chains, so I based everything on that, `isNative`, `!isNative`.